### DISCUSSION TITLE
Support built-in RSS feed generation

### DISCUSSION THREAD

**Discussion by ovflowd** [shuding/nextra#1685] [Ideas]
I'd love if Nextra could support built-in RSS feed generation. Mostly because Nextra already built the MDX content into HTML during its compile process.

On our tests [example PR here](https://github.com/nodejs/nodejs.org/pull/5144), adding outside RSS generation would require us to build the whole MDX content twice.

**Comment by dimaMachina** [comment]
Hi, I try to help you first this week and maybe integrate rss generation directly in Nextra
